python/Dockerfile.python

```diff
@@ -1,11 +1,15 @@
-FROM ghcr.io/epfl-lasa/control-libraries/development-dependencies:latest as python-install
+ARG BASE_TAG=latest
+FROM ghcr.io/epfl-lasa/control-libraries/development-dependencies:${BASE_TAG} as source
 ARG BRANCH=develop
 
 WORKDIR /source
 RUN git clone --depth 1 --branch ${BRANCH} https://github.com/epfl-lasa/control-libraries
```

Comment:
This (together with the demos Dockerfile) makes me concede that it might be nice to have a hosted image with the full prebuilt library binaries, on 20.04 and 22.04 for amd64 and arm64. It would have to be rebuilt on push to main (for [:22.04, :latest], and :20.04 tags) and push to develop (for [:22.04-devel, :devel] and :20.04-devel tags). It would essentially be what the ros2_control_libraries layer does in aiea-technology/docker-images, and in turn would save the need to run the installation in that layer. 

But, it's absolutely not a priority and I think these locally built images (demos, python, etc) can fully handle the install script being run once in a while. I think our image pipeline has had enough rework for the short term!

Reply:
I added this to our infinity backlog